docs/graphcache/gotchas.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gotchas
+order: 5
+---
+
+# Gotchas
+
+When using the normalised cache there are a few out of the box features
+we need to be aware of, being aware of these features will make it easier
+to define best practices and to avoid pitfalls.
+
+Most of the measures we'll cover are brought to life to ensure we can build out
+to a fully offline normalised cache.
+
+## Commutativity
+
+By default the cache ensures idempotent and commutative results, the last part of this
+is the most relevant one to be aware of.
```

Comment:
```suggestion
By default the cache ensures that results from the API are applied commutatively.
```

- removing `idempotent` as it's technically not reapplying results, so this doesn't apply
- "last part" is ambiguous but also not needed anymore
- fixes comma separating two distint sentences